shell/browser/electron_browser_main_parts.cc

```diff
@@ -451,7 +451,8 @@ void ElectronBrowserMainParts::PreMainMessageLoopRun() {
   auto* command_line = base::CommandLine::ForCurrentProcess();
   if (command_line->HasSwitch(switches::kRemoteDebuggingPipe)) {
     // --remote-debugging-pipe
-    content::DevToolsAgentHost::StartRemoteDebuggingPipeHandler();
+    content::DevToolsAgentHost::StartRemoteDebuggingPipeHandler(
+        base::OnceClosure());
```

Comment:
Shouldn't we also close the app when the pipe disconnects ? Any automated driver that starts electron with `--remote-debugging-pipe` would like that feature I guess ?

Reply:
Good call!

https://github.com/electron/electron/pull/26535/commits/fc493cc305e9f8b66ace9488008619b84e34a40a